src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -136,4 +136,14 @@ A command is running to initially populate your local package cache, to improve
   <data name="FailedToPrimeCacheError" xml:space="preserve">
     <value>Failed to prime the NuGet cache. {0} failed with: {1}</value>
   </data>
+  <data name="UnauthorizedAccessMessage"  xml:space="preserve">
+    <value>Permission denied to modify the '{0}' folder.
+
+Here are some options to fix this error:
+---------------------
+1. Re-run this command with elevated access.
+2. Disabled the first run experience by setting the environment variable DOTNET_SKIP_FIRST_TIME_EXPERIENCE to true.
+3. Copy the .NET Core SDK to a non-protected location and use it from there.
```

Comment:
They can't get into this situation unless they are using a zip already. If a native installer is used, it will run the first run experience implicitly.